handlers/jaeger.go

```diff
@@ -34,7 +28,7 @@ func GetJaegerInfo(w http.ResponseWriter, r *http.Request) {
 func GetJaegerServices(w http.ResponseWriter, r *http.Request) {
 	business, err := getBusiness(r)
 	if err != nil {
-		RespondWithError(w, http.StatusInternalServerError, "Services initialization error: "+err.Error())
+		RespondWithError(w, http.StatusInternalServerError, "Token initialization error: "+err.Error())
```

Comment:
Not sure why it's changed to `Token` (here and below), the function `getBusiness` may fail in case of client initialization error for instance, it's not necessarily about the token

Reply:
You're right I'll change this